lib/sacastats_web/templates/character/weapons.html.heex

```diff
@@ -74,7 +74,7 @@
                         <img src={"https://census.daybreakgames.com/files/ps2/images/static/" <> (SacaStatsWeb.CharacterController.get_medal_code(SacaStatsWeb.CharacterController.get_total_values(Map.get(weapon, "weapon_kills"), @character["response"]["faction_id"]), 
```

Comment:
It doesn't look like it's letting me suggest a change there, but I if you also add some padding for the weapon column header then it will look better for mobile. It looks like this:

`<th data-field="weapon" data-toggle="tooltip" title="Weapons" class="pr-5">Weapon</th>`

`pr-5` is a bootstrap class and it's the biggest padding-right that it has. I think we might want to go a bit bigger with a custom padding-right class in the `weapons.css` stylesheet. But I'm not sure exactly what that value would be.